### PR TITLE
Fix tablet showing up for a short period of time on launch

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

**Comment by huffman** [comment]
The goal of this PR is to fix an issue where the tablet shows up on launch of a short period of time (0-2 seconds, or so).

To test:

 1. Launch interface
 1. Pay close attention to whether the tablet shows up. On master you should see it show up. On this build you should not
  1. Test this on both Desktop and HMD
  1. Test this when you launch into the tutorial
 1. Test that you can open and close the tablet as usual

**Comment by sethalves** [comment]
When you first summon the tablet (on purpose) does it appear quickly?  The reason it show up like this is so that it's ready to go when summoned.

**Comment by huffman** [comment]
@sethalves I can test that

**Comment by huffman** [comment]
Definitely has a delay. I'll look into preloading the necessary resources.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 51db0cef98138a58e6ab60ada58d5ecb706c3acc available at: https://gist.github.com/52986a1a1466a9c9ba75ad47a261576a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10426-51db0ce.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10426-51db0ce.dmg

**Comment by huffman** [comment]
@sethalves Ok, I modified Web3DOverlay so that it always preloads QML files even when not visible. The cpu overhead should be minimal after having loaded the QML and not drawing. Open to feedback.

**Comment by hifi-gustavo** [comment]
Build console output for commit: bddfdfa838d66a3384e55d1d5a4ab9fd485722c7 available at: https://gist.github.com/52986a1a1466a9c9ba75ad47a261576a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10426-bddfdfa.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10426-bddfdfa.dmg

**Comment by PhilipRosedale** [comment]
QA Results:  PASS
Tested many times starting up in tutorial and different locations.  Tablet does not appear when it should not, and is responsive coming up when I invoke it.

File: scripts/system/libraries/WebTablet.js [hunk 1]
```diff
@@ -107,6 +107,8 @@ WebTablet = function (url, width, dpi, hand, clientOnly, location) {
     this.depth = TABLET_NATURAL_DIMENSIONS.z * tabletScaleFactor;
     this.landscape = false;
 
+    visible = visible === true;
```
Comment (thoys) [inline scripts/system/libraries/WebTablet.js]: You're just making sure it is a boolean that comes in, I can see that

File: interface/src/ui/overlays/Web3DOverlay.cpp [hunk 2]
```diff
@@ -282,6 +278,10 @@ void Web3DOverlay::render(RenderArgs* args) {
         _webSurface->resize(QSize(_resolution.x, _resolution.y));
     }
 
+    if (!_visible || !getParentVisible()) {
+        return;
+    }
```
Comment (thoys) [inline interface/src/ui/overlays/Web3DOverlay.cpp]: You moved this check to here to create the Web3DOverlay even though it is not visible, to preload the content, looks good.

**Pull request merged**